### ISSUE TITLE
Can't export project, unknown modules

### ISSUE THREAD

**Issue by GamaDu30** [godotjs/GodotJS#11]
OS: Windows 11
Godot Version: 4.3.stable (custom build with GodotJS)

When we try to run the exported project, An error appears:

```
WARNING: [jsb][Warning] something went wrong on loading 'res://.godot/GodotJS/DE/nodes/DreamEngineMainNode.js'
Uncaught Error: unknown module: eventemitter3
Error: unknown module: eventemitter3
    at <anonymous>:19:41
    at <anonymous>:7:38
    at <anonymous>:14:36
     at: jsb::Environment::load (modules\GodotJS\bridge\jsb_environment.cpp:894)
ERROR: [jsb][Error] failed to attach module res://.godot/GodotJS/DE/nodes/DreamEngineMainNode.js (36)
   at: GodotJSScript::load_module_immediately (modules\GodotJS\weaver\jsb_script.cpp:457)
```

We suspect that it might be connected to the transpilation process not bundling "node_modules" files, but we could have missed something while building our Godot and Godot export template.

Do we need to rebuild modules ourselves from scratch directly in our code base or is it possible to import them ?

Please feel free to ask if you need more details about our setup or Godot compilation options.

**Comment by ialex32x** [comment]
I'll investigate it. Could you attach your `package.json` here and any details about how `eventemitter3` is imported in `DreamEngineMainNode`?

**Comment by GamaDu30** [comment]
Hey, thanks for the quick answer !

[Here is our package.json](https://github.com/user-attachments/files/17408430/package.json)

eventEmitter3 is a npm package that we included in our package.json file and installed with "npm i"
And then it's imported like this in our files:
`import EventEmitter from 'eventemitter3';`

The commit we used to build Godot is:
`77dcf97d82cbfe4e4615475fa52ca03da645dbd8`
And the commit of GodotJS is:
`d382fdddeb4055a765be61a6f1c8f262ccf8723f`

**Comment by ialex32x** [comment]
Oops, I missed something about how a module from `node_modules` is loaded in the exported project (the package.json of npm packages are not exported). I'll work on resolving it soon.  

By the way, the support for npm packages is still very limited, so it's best to avoid using them for now.

**Comment by GamaDu30** [comment]
Ok no problem ! eventEmitter3 was the only package from node_modules that we were using, so we will be able to find a solution without using code from node_modules.

**Comment by GeTechG** [comment]
I'm not sure if this solution fits the problem, but sometimes it has helped me to rebuild the library manually via webpack using commonjs as the target.

**Comment by ialex32x** [comment]
It's fixed in 3fd9d04 (at least for eventemitter3).
